src/js/actions/layereffects.js

```diff
@@ -325,6 +333,89 @@ define(function (require, exports) {
     setShadowSpread.reads = [locks.PS_DOC, locks.JS_DOC];
     setShadowSpread.writes = [locks.PS_DOC, locks.JS_DOC];
 
+    /**
+     * Copies the layer effects of the source layer to all the layers
```

Comment:
I think transfer is more wrong than copy, since they remain at the source layer as well.